subprojects/core/src/integTest/groovy/org/gradle/api/ApplyPluginIntegSpec.groovy

```diff
@@ -199,6 +199,10 @@ class ApplyPluginIntegSpec extends AbstractIntegrationSpec {
                 implementation gradleApi()
                 implementation localGroovy()
             }
+
+            test {
+                useJUnit()
+            }
```

Comment:
Shouldn't the default still be JUnit?

Reply:
Yes, I think was an intermediate change I did while trying to fix tests. I think it can be reverted.